modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +62,16 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const copyAppropriateAttributes = <T extends HTMLElement> (original: SugarElement<T>, clone: SugarElement<T>): void => {
+  Arr.each(Obj.keys(transferableAttributes), (attributeName: string) => {
+    const attribute = Attribute.get(original, attributeName);
+
+    if (Type.isString(attribute) && Arr.contains(transferableAttributes[attributeName], attribute)) {
+      Attribute.set(clone, attributeName, attribute);
+    }
```

Comment:
This should be using `Attribute.getOpt` instead of the current get + type check.
```suggestion
    Attribute.getOpt(original, attributeName)
      .filter((attribute) => Arr.contains(transferableAttributes[attributeName], attribute))
      .each(attribute) => Attribute.set(clone, attributeName, attribute));
```

Reply:
I'm honestly not sure at all how this is any cleaner than what we already have, but sure.